poleno/attachments/migrations/0003_content_type.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import magic
+
+from django.db import models, migrations
```

Comment:
Importy zvykneme rozdelovat do dvoch skupin:

- naprv built in python importy
- potom importy z doinstalovanych kniznic
- potom importy zo suborov lokalnych k projektu s absolutnou cestou
- potom importy z aktualneho adresara

Jednotlive skupiny oddelujeme prazdnym riadkom. Cize v tomto pripade:

```
from __future__ import unicode_literals

import magic
from django.db import models, migrations 
```